calico-cloud/image-assurance/install-the-admission-controller.mdx

```diff
@@ -61,36 +61,44 @@ Container admission policies are custom Kubernetes resources that allow you to c
 
    :::caution
 
-   If you generate the key and certificate pair yourself, you must set the SANS to `tigera-image-assurance-admission-controller-service.tigera-image-assurance.svc`
+   If you generate the key and certificate pair yourself, you must set the SANS to `tigera-image-assurance-admission-controller-service.tigera-image-assurance.svc`.
 
    :::
 
 1. Download and configure the Admission Controller manifests.
 
-   As a safety mechanism, we require that you specify the namespaces that the Admission Controller will apply in
-   the `IN_NAMESPACE_SELECTOR_KEY` and the `IN_NAMESPACE_SELECTOR_VALUES`. These values configure the Kubernetes API server to send admission requests to our Admission Controller only for resources from relevant namespaces.
+As a safety mechanism, we require that you specify the namespaces that the Admission Controller will apply in
+the `IN_NAMESPACE_SELECTOR_KEY` and the `IN_NAMESPACE_SELECTOR_VALUES`. These values configure the Kubernetes API server to send admission requests to our Admission Controller only for resources from relevant namespaces.
 
-   For example, to configure the Kubernetes API server to send admission requests for resources created in any namespace with label key `foo`, and label values either `bar` or `baz`, the variables would be set as follows:
+For example, to configure the Kubernetes API server to send admission requests for
+resources created in any namespace with label key `name`, and label values either `prod` or `staging`, set the variables as follows:
 
    ```bash
-   export IN_NAMESPACE_SELECTOR_KEY="foo" && IN_NAMESPACE_SELECTOR_VALUES="bar baz"
+   export IN_NAMESPACE_SELECTOR_KEY="name" && IN_NAMESPACE_SELECTOR_VALUES="prod staging"
    ```
+Here is the namespace manifest with the staging label and name key.
+
+   ```yaml
+   kind: Namespace
+   apiVersion: v1
+   metadata:
+     name: production
```

Comment:
let's change this name here to staging-test

Reply:
Fixed.

Reply:
Sorry, The `name: production` field should also be `name: staging-test`, after that it looks good to me.

Reply:
Fixed.